samples/interactive-tutorials/README.md

```diff
@@ -99,16 +92,6 @@ To run a code sample from the Cloud Shell, you need to be authenticated using th
     export GOOGLE_APPLICATION_CREDENTIALS=~/key.json
```

Comment:
Aren't we trying to de-emphasize GOOGLE_APPLICATION_CREDENTIALS in favor of `gcloud auth application-default login` ?

Reply:
using  GOOGLE_APPLICATION_CREDENTIALS to request the Retail servicei is a recommented way to do that.

Reply:
Really?  Most of the time we are providing you with credentials. 
* https://google.aip.dev/auth/4110

Reply:
this is discribed in the Retail documentation we are relying creating this tutorials https://cloud.google.com/retail/docs/setting-up#local-environment

Reply:
That isn't the way things are supposed to be these days.

Reply:
Can you reach out to a TechWriter?

Reply:
https://cloud.google.com/docs/authentication/production#automatically
From our previous experiences, gcloud CLI often results in using end user credentials instead of the service account credentials in some languages, and causing errors like "Your application has authenticated using end user credentials from the Google Cloud SDK or Google Cloud Shell which are not supported."

In this case, we prefer to have users explicitly set GOOGLE_APPLICATION_CREDENTIALS so that end user credentials won't be used.